tests/e2e/utils/DriverHelper.ts

```diff
@@ -706,6 +706,10 @@ export class DriverHelper {
         await this.enterValue(elementLocator, value, timeout);
     }
 
+    public async highLightElement(element: WebElement): Promise<void> {
```

Comment:
1) "public" is redundant in method declaration
2) I couldn't find the method consumer in PR; is the method really needed?

Reply:
This method not mandatory but very useful for page-objects debugging. Can we keep it somehow?

Reply:
public removed